file_formats/cmt_format.ml

```diff
@@ -99,6 +104,17 @@ let iter_on_annots (it : Tast_iterator.iterator) = function
   | Partial_implementation array -> Array.iter (iter_on_parts it) array
   | Partial_interface array -> Array.iter (iter_on_parts it) array
 
+module Local_reduce = Shape.Make_reduce(struct
```

Comment:
We now have two `Local_reduce` in the compiler: `Shape.Local_reduce` and this one.
Could you document how they differ? (Could this difference be reflected in the name?)

Reply:
I renamed the one in Shapes and added more information on its usage's restrictions in 1d5b0a3